rust/actyx/swarm/cli/src/main.rs

```diff
@@ -25,25 +25,50 @@ use tokio::{
     runtime::Handle,
     sync::mpsc,
 };
+use tracing_subscriber::{fmt::format::FmtSpan, EnvFilter};
 use trees::{query::TagExprQuery, AxKey};
 use util::variable::Writer;
 
+fn make_log_filename() -> String {
+    std::env::vars()
+        .find(|x| x.0 == "NETSIM_TEST_LOGFILE")
+        .map(|x| x.1)
+        .unwrap_or("unknown".to_string())
```

Comment:
this could be simplified to `std::env:var("NETSIM_TEST_LOGFILE").unwrap_or_else(|_| "unknown".to_owned())`

Reply:
I'll include the fix on my next PR